### PR TITLE
Introducing a new script to copy USWDS tokens

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://asteele-passthrough-tokens-1967--60f9b557105290003b387cd5.chromatic.com

---

## Description
Adds a script for copying over desired USWDS tokens into the CSS Library package. The list of desired tokens is taken from this [VA Design System Airtable](https://airtable.com/appF4FJKBeNVNGrlm/tblaTo3cDAURdk22q/viwkqvCZIOuv2RT6P?blocks=hide).

Closes #[1967](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1967)

## Testing done
Locally tested.

## Screenshots
Example of changing `color-black` to look to the USWDS color:
Screenshot 2023-08-23 at 11 09 32 AM
Screenshot 2023-08-23 at 11 08 30 AM

## Acceptance criteria
- [X] Tokens import and can be used as variables.

## Definition of done
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/css-library/copy-uswds-color-tokens.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    file: 'violet.json',
+    colors: ['violet-vivid-70'],
+  },
+  'yellow': {
+    file: 'yellow.json',
+    colors: ['yellow-5'],
+  },
+};
+// Desired prefix/namespace for colors
+const prefix = 'uswds-system-color';
```
Comment (jamigibbs) [inline packages/css-library/copy-uswds-color-tokens.js:96]: Is this a specific prefix that we need to use? Or just a general way to identify those tokens as "uswds"? Just a nit but could this just be shortened to `uswds`?  It just seems like some of the tokens get really long with it: 

`--uswds-system-color-blue-warm-vivid-80: #162e51;`

File: packages/css-library/dist/tokens/css/variables.css [hunk 2]
```diff
@@ -102,4 +102,57 @@
...
+  --uswds-system-color-gray-5: #f0f0f0;
+  --uswds-system-color-gray-10: #e6e6e6;
+  --uswds-system-color-gray-20: #c9c9c9;
+  --uswds-system-color-gray-30: #adadad;
+  --uswds-system-color-gray-40: #919191;
+  --uswds-system-color-gray-50: #757575;
+  --uswds-system-color-gray-60: #5c5c5c;
+  --uswds-system-color-gray-70: #454545;
+  --uswds-system-color-gray-80: #2e2e2e;
+  --uswds-system-color-gray-90: #1b1b1b;
```
Comment (jamigibbs) [inline packages/css-library/dist/tokens/css/variables.css:139]: This is a duplicate color with one that we had to manually add previously. Should be go ahead and just remove the duplicates while we're at it here? 

 `--v3-color-base-darkest: #1B1B1B;`

Comment (jamigibbs) [reply packages/css-library/dist/tokens/css/variables.css:139]: There is also one other we manually added (it's that error red color `red-60v`):  

`--v3-color-error-dark: #b50909;`

https://github.com/department-of-veterans-affairs/component-library/blob/17280b3152a4b235fed72f6b610b24bf05a09e36/packages/css-library/dist/tokens/css/variables.css#L49C7-L50

File: packages/css-library/copy-uswds-color-tokens.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  .reduce((accumulator, prop) => {
+    accumulator[prop.name] = { value: prop.value };
+    return accumulator;
+  }, {});
+
+// Last, need to write out uswdsTokens into a tokens/uswds.json file
+let data = JSON.stringify(uswdsTokens, null, 2);
+fs.writeFileSync('./tokens/uswds.json', data);
+
+console.log('USWDS Tokens copied successfully.');
```
Comment (jamigibbs) [inline packages/css-library/copy-uswds-color-tokens.js:169]: Intentional for terminal logging?

File: packages/css-library/copy-uswds-color-tokens.js (hunk 1 above)
Comment (Andrew565) [reply packages/css-library/copy-uswds-color-tokens.js:96]: This is the prefix that @humancompanion-usds indicated he wanted to use, but easy to change it if we want to.

File: packages/css-library/copy-uswds-color-tokens.js (hunk 3 above)
Comment (Andrew565) [reply packages/css-library/copy-uswds-color-tokens.js:169]: Yes, intentional for terminal logging.

File: packages/css-library/dist/tokens/css/variables.css (hunk 2 above)
Comment (jamigibbs) [reply packages/css-library/dist/tokens/css/variables.css:139]: @Andrew565 Before merging, do you mind just linking to the follow-up ticket for this here? 🙏

Comment (Andrew565) [reply packages/css-library/dist/tokens/css/variables.css:139]: Follow-up ticket to replace color tokens with variables wherever possible: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2037

**Review by jamigibbs — APPROVED** [review]
Thank you! ✨

**Pull request merged**